src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -49,6 +51,8 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.__maintenance_window_exceeded = False
         self.__installation_reboot_status = Constants.RebootStatus.NOT_NEEDED
         self.__installation_packages_map = collections.OrderedDict()
+        self.__installation_substatus_msg_copy = None
+        self.__installation_packages_copy = []
```

Comment:
Leave a general comment in code next to the variables (once per area - so line 55, 65, 116) stating why a copy is needed and present. Look at it from the lens of someone who wasn't part of this PR and is looking at all this code in master in the future.

Reply:
addressed

Reply:
Checked and resolved.